pkg/handlers/generic/lifecycle/cni/cilium/handler.go

```diff
@@ -183,6 +185,33 @@ func (c *CiliumCNI) apply(
 			helmValuesSourceRefName = cniVar.Values.SourceRef.Name
 			// Use cluster's namespace since Values.SourceRef is always a LocalObjectReference
 			targetNamespace = cluster.Namespace
+
+			err := handlersutils.EnsureClusterOwnerReferenceForObject(
```

Comment:
Adding ownership here will delete the configmap when the cluster is deleted. I'm not sure that is what we want to do as the configmap is user-created, not controller created.

Reply:
This is the same behavior we already have with credentials Secrets and decided then to go with ownerRefs.

Reply:
OK.